### ISSUE TITLE
bug: YouTube immediately crashes after patching

### ISSUE THREAD

**Issue by NMBlanco** [inotia00/ReVanced_Extended#2521]
### Type

Error at runtime

### Tools used

RVX Manager

### Application

19.16.39

### Bug description

When the patching is finished (without any errors) and I click mount, then open YouTube, it immediately crashes.
Using latest 5.0.1 patches.

### Device Environment

Android 14

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I did not use any settings marked as `Experimental Flags`.
- [X] I have patched the APK according to the [documentation](https://github.com/inotia00/revanced-documentation#readme).
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by inotia00** [comment]
Please attach crash log

**Comment by NMBlanco** [comment]
> Please attach crash log

[log_2024_12_08_15_09_04.log](https://github.com/user-attachments/files/18052559/log_2024_12_08_15_09_04.log)

**Comment by inotia00** [comment]
Just a while ago I also patched and mounted YouTube 19.16.39 with RVX Manager

The issue is not reproduced on my side

Also the crash log points to the `M` method of the `ccw` class, but no patch was applied to this method

Can you provide a list of patches that are included or excluded?

**Comment by NMBlanco** [comment]
> Just a while ago I also patched and mounted YouTube 19.16.39 with RVX Manager
> 
> The issue is not reproduced on my side
> 
> Also the crash log points to the `M` method of the `ccw` class, but no patch was applied to this method
> 
> Can you provide a list of patches that are included or excluded?

Hm weird... 
Here are the patches I selected:
```
- Device Info
RVX Manager: 1.23.4
Model: motorola edge 30 ultra
Android version: 14
Supported architectures: arm64-v8a, armeabi-v7a, armeabi
Root permissions: Yes

- Patch Info
App: com.google.android.youtube v19.16.39 (Suggested: 19.16.39)
Patches version: v5.0.1
Patches added: Force hide player buttons background, Hide Shorts dimming, Return YouTube Username
Patches removed: Hook YouTube Music actions, Custom Shorts action buttons, Custom branding name for YouTube, Translations for YouTube, Enable debug logging, Overlay buttons, Swipe controls, GmsCore support
Default patch options changed: None

- Settings
Allow changing patch selection: true
Version compatibility check: false
Show universal patches: false
Remove unused library: false
Use pre-release: true
Patches source: inotia00/revanced-patches

- Logs
Reading APK
Decoding app manifest
Loading patches
Deleting existing temporary files directory
Decoding resources
Initializing lookup maps
Executing patches
Applied 51 patches
Compiling modified resources
Aligning APK
Signing APK
Patched APK
Mounting patched app
Installed
```

**Comment by inotia00** [comment]
I re-patched the same patch list you provided and was able to reproduce the issue on my side

**Comment by inotia00** [comment]
It looks like the `Overlay buttons` patch is causing the issue

Would you please include this patch and try again?

**Comment by NMBlanco** [comment]
> It looks like the `Overlay buttons` patch is causing the issue
> 
> Would you please include this patch and try again?

Yes with this patch enabled I got it to work. No crash anymore. Thanks!

**Comment by inotia00** [comment]
fixed in [revanced-patches-5.0.2](https://github.com/inotia00/revanced-patches/releases/tag/v5.0.2)
